assets/scss/custom.scss

```diff
@@ -323,7 +323,6 @@ footer {
   table {
     table-layout: fixed;
     width: 100%;
-    margin: 4rem 0;
```

Comment:
I did this because otherwise I ended up with a huge gap of whitespace above the new tables. I can't find anywhere else that it has changed anything.

Reply:
See the 200 response of https://pr1166--matrix-spec-previews.netlify.app/client-server-api/#getwell-knownmatrixclient for an example where this merges the tables into two. 

(xref: https://github.com/matrix-org/matrix-spec/commit/ea42cd3c7b40caae0b6925e1620173dd004e04a7#r79081346)

Reply:
Here's a quick prototype. WDYT?

![image](https://user-images.githubusercontent.com/8614563/180266911-2943828c-5373-4c90-b4da-15c7782d745c.png)

Reply:
sure, lgtm!

Reply:
new PR incoming!